bobtemplates/odoo/model/views/+model.name_underscored+.xml.bob

```diff
@@ -75,7 +75,6 @@
         <field name="name">{{{ model.name_camelwords }}}</field> <!-- TODO -->
         <field name="res_model">{{{ model.name_dotted }}}</field>
         <field name="view_mode">tree,form</field>
-        <field name="view_type">form</field>
```

Comment:
I suggest to keep this field. Ok, it's the defaut value but at least tagging is ready to be modified if any. It should be nice to do the same for `context` and `domain`. No ?